mongodbatlas/advanced_clusters.go

```diff
@@ -0,0 +1,230 @@
+	BackingProviderName string `json:"backingProviderName,omitempty"`
+	Priority            int    `json:"priority,omitempty"`
+	ProviderName        string `json:"providerName,omitempty"`
+	RegionName          string `json:"regionName,omitempty"`
+}
+
+type AdvancedAutoScaling struct {
+	DiskGB  *DiskGB  `json:"diskGB,omitempty"`
+	Compute *Compute `json:"compute,omitempty"`
+}
+
+type DiskGB struct {
+	Enabled *bool `json:"enabled,omitempty"`
+}
+
+type Specs struct {
+	DiskIOPS      float64 `json:"diskIOPS,omitempty"`
+	EbsVolumeType string  `json:"ebsVolumeType,omitempty"`
+	InstanceSize  string  `json:"instanceSize,omitempty"`
+	NodeCount     int     `json:"nodeCount,omitempty"`
+}
+
+// AdvancedClustersResponse is the response from the AdvancedClustersService.List.
+type AdvancedClustersResponse struct {
+	Links      []*Link            `json:"links,omitempty"`
+	Results    []*AdvancedCluster `json:"results,omitempty"`
+	TotalCount int                `json:"totalCount,omitempty"`
+}
+
+// List all clusters in the project associated to {GROUP-ID}.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/clusters-get-all/
+func (s *AdvancedClustersServiceOp) List(ctx context.Context, groupID string, listOptions *ListOptions) (*AdvancedClustersResponse, *Response, error) {
+	if groupID == "" {
+		return nil, nil, NewArgError("groupId", "must be set")
+	}
+	path := fmt.Sprintf(advancedClustersPath, groupID)
+
+	// Add query params from listOptions
+	path, err := setListOptions(path, listOptions)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.Client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new(AdvancedClustersResponse)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	if l := root.Links; l != nil {
+		resp.Links = l
+	}
+
+	return root, resp, nil
+}
+
+// Get gets the cluster specified to {ClUSTER-NAME} from the project associated to {GROUP-ID}.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/clusters-get-one/
+func (s *AdvancedClustersServiceOp) Get(ctx context.Context, groupID, clusterName string) (*AdvancedCluster, *Response, error) {
+	if groupID == "" {
+		return nil, nil, NewArgError("groupId", "must be set")
+	}
+	if err := checkClusterNameParam(clusterName); err != nil {
+		return nil, nil, err
+	}
+
+	basePath := fmt.Sprintf(advancedClustersPath, groupID)
+	escapedEntry := url.PathEscape(clusterName)
+	path := fmt.Sprintf("%s/%s", basePath, escapedEntry)
+
+	req, err := s.Client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new(AdvancedCluster)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return root, resp, err
+}
+
+// Create adds a cluster to the project associated to {GROUP-ID}.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/clusters-create-one/
```

Comment:
copy/paste?
```suggestion
// See more: https://docs.atlas.mongodb.com/reference/api/cluster-advanced/create-one-cluster-advanced/
```

same for other doc links